File: trunk/theLink/example/vb/MyClient.vb
REM
REM  \file       theLink/example/vb/MyClient.vb
REM  \brief      \$Id$
REM  
REM  (C) 2009 - NHI - #1 - Project - Group
REM  
REM  \version    \$Rev$
REM  \author     EMail: aotto1968 at users.sourceforge.net
REM  \attention  this software has GPL permissions to copy
REM              please contact AUTHORS for additional information
REM

Imports System
Imports csmsgque

Public Module example
    Sub Main(ByVal args() As String)
        Dim c As New MqS()
        Try
            c.LinkCreate(args)
	    c.SendSTART()
	    c.sendEND_AND_WAIT("HLWO", 5)
	    Console.WriteLine(c.ReadC())
        Catch ex As Exception
            c.ErrorSet(ex)
        Finally
            c.Exit()
        End Try
    End Sub
End Module

File: trunk/theLink/example/csharp/MyClient.cs
/**
 *  \file       example/csharp/MyClient.cs
 *  \brief      \$Id: MyClient.cs 507 2009-11-28 15:18:46Z dev1usr $
 *  
 *  (C) 2009 - NHI - #1 - Project - Group
 *  
 *  \version    \$Rev: 507 $
 *  \author     EMail: aotto1968 at users.sourceforge.net
 *  \attention  this software has GPL permissions to copy
 *              please contact AUTHORS for additional information
 */
using System;
using csmsgque;

namespace example {

  sealed class MyClient : MqS {

    static void Main(string[] argv) {
      MyClient c = new MyClient();
      try {
	c.LinkCreate(argv);
	c.SendSTART();
	c.SendEND_AND_WAIT("HLWO", 5);
	Console.WriteLine(c.ReadC());
      } catch (Exception ex) {
        c.ErrorSet (ex);
      }
      c.Exit();
    }
  }
}
